cheeseshop/dbapi.py

```diff
@@ -323,8 +357,18 @@ class CsGoDeathEvent(object):
             CREATE TABLE cs_go_death_events(
                 id serial PRIMARY KEY,
                 attacker integer REFERENCES cs_go_steam_ids (id),
+                attacker_pos_x real,
+                attacker_pos_y real,
+                attacker_pos_z real,
                 victim integer REFERENCES cs_go_steam_ids (id),
-                weapon text
+                victim_pos_x real,
+                victim_pos_y real,
+                victim_pos_z real,
+                assister integer REFERENCES cs_go_steam_ids (id),
+                weapon_original_owner integer REFERENCES cs_go_steam_ids (id),
+                penetrated boolean,
+                weapon text,
+                map text
```

Comment:
Should we also keep track of the game ID and the tick here? That would make it easy to find the first player_death event of any game.

Reply:
Yah we need meta information for at least:

match
map (sometimes matches are best of 3 or best of 5)
round
tick
seconds into round?